crates/bevy_color/src/hsva.rs

```diff
@@ -157,7 +157,11 @@ impl From<Hwba> for Hsva {
     ) -> Self {
         // Based on https://en.wikipedia.org/wiki/HWB_color_model#Conversion
         let value = 1. - blackness;
-        let saturation = 1. - (whiteness / value);
+        let saturation = if value != 0. {
+            1. - (whiteness / value)
+        } else {
+            0.
+        };
```

Comment:
Could you used `checked_div` for this?
```suggestion
        let saturation = 1. - whiteness.checked_div(value).unwrap_or(1.);
```

Reply:
yep same behaviour in both, definite improvment.

Reply:
Oh wait no such method?

Reply:
Oops my bad. I could've sworn it existed for floats too 😅